djstripe/fields.py

```diff
@@ -174,7 +174,7 @@ def __init__(self, enum, *args, **kwargs):
         super(StripeEnumField, self).__init__(*args, **defaults)
 
     def deconstruct(self):
-        name, path, args, kwargs = super().deconstruct()
+        name, path, args, kwargs = super(StripeEnumField, self).deconstruct()
```

Comment:
I included these as I deliberately branched from stable/1.2.0 because it's the last version that supports Django 1.11. As you said you're going to support 1.2.0 until Django 1.11 reaches end of life and dj-stripe 1.2.0 supports Python 2.7 wouldn't we need to include them?

Reply:
master is the 1.3.0 branch. For 1.2.0 we'll cherry-pick from master onto stable/1.2

Reply:
Yeh but I deliberately branched from stable/1.2 so as to maintain the Dj 1.11 compatibility

Reply:
Yeh but I deliberately didn't branch from master so as to maintain Dj 1.11 compatibility. My branch was split from stable/1.2 and that's what the pull request is for - to merge my changes into stable/1.2, hence the python 2.7 tweaks.

Reply:
I see. Cherry picking this into 1.2 will not be an issue, but this has to land on master as well.

Can you instead make a separate PR for any Python 2.7 fix needed on stable/1.2, and move this current PR to master (or file a new one if that's not possible -- I dont think you can move branches within PRs). I'll take care of cherry-picking this into stable/1.2.